rules/Defluent/ValueObject/AssignAndRootExpr.php

```diff
@@ -17,7 +17,7 @@ final class AssignAndRootExpr extends AbstractRootExpr implements RootExprAwareI
     public function __construct(
         Expr $assignExpr,
         Expr $rootExpr,
-        private ?Variable $silentVariable = null,
+        private ?Variable $variable = null,
```

Comment:
it seems need check on nullable as well.

Reply:
fixed https://github.com/rectorphp/rector-src/pull/168/commits/0879f29f8dfdfe97d2a9451516a7d8342c961b2e